apps/src/gamelab/GameLabSprite.js

```diff
@@ -238,7 +238,7 @@ module.exports.createSprite = function (x, y, width, height) {
    // frame, which will cause the animation to continue on the next update().
    // If the animation is non-looping and is stopped at the last frame
    // we also rewind the animation to the beginning.
-    if (!s.animation.looping && s.animation.getFrame() === s.animation.images.length - 1) {
+    if (!s.animation.looping && !s.animation.playing && s.animation.getFrame() === s.animation.images.length - 1) {
```

Comment:
Good test coverage.  Did you check to see if we can now remove the `!s.animation.looping` check here?  I think it's redundant now.